defaultBookshelfRepository.js

```diff
@@ -104,15 +104,14 @@ const paginate = (options = {}) => {
 };
 
 const count = (options = {}, model) => {
-    if (!model) {
+    if (!options.count) return qb => qb;
```

Comment:
how do these changes contribute to the requested functionality?

Reply:
To maintain compatibility with `query` method for collection usage